website/server/controllers/api-v3/challenges.js

```diff
@@ -200,6 +205,7 @@ api.createChallenge = {
     const { user } = res.locals;
 
     req.checkBody('group', apiError('groupIdRequired')).notEmpty();
+    if (req.body.challenge.summary && req.body.challenge.summary.length > MAX_SUMMARY_SIZE_FOR_CHALLENGES) throw new BadRequest(res.t('summaryTooLong'));
```

Comment:
what if `req.body.challenge` is not defined? Also this cose should go after the validation errors